packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethods.js

```diff
@@ -36,37 +36,48 @@ const PaymentMethods = props => {
         return null;
     }
 
-    const radios = availablePaymentMethods.map(({ code, title }) => {
-        // If we don't have an implementation for a method type, ignore it.
-        if (!Object.keys(payments).includes(code)) {
-            return;
-        }
+    const radios = availablePaymentMethods
+        .map(({ code, title }) => {
+            // If we don't have an implementation for a method type, ignore it.
+            if (
+                !Object.keys(PAYMENT_METHOD_COMPONENTS_BY_CODE).includes(code)
```

Comment:
Where does `PAYMENT_METHOD_COMPONENTS_BY_CODE` come from?

Reply:
Sorry i copied it from the released package. I am reverting it back